### ISSUE TITLE
Add CSV lint documentation

### ISSUE THREAD

**Issue by dicknorris** [theodi/shared#683]
https://github.com/theodi/csvlint/issues/142

"It would be nice to have some documentation, at least one easy/minimal walk-through, how to deploy it, e.g. on heroku, and/or a plain linux, and/or vagrant...

It is a very interesting approach and we really would like to contribute!"

Update 18/08/2015 by @bcouston:

Meant for a mix of users and developers. Documentation to be split accordingly:

- About -> Non-technical information for users

- Documentation -> Technical information ad walkthrough for developers (including webservice API)

**Comment by bcouston** [comment]
@dicknorris and @floppy, we have discussed in the past about how validations and data is stored depending on whether you input a URL or a file, can one you guys reiterate how this is handled in each case and whether this should be added to the "About" page in a privacy section or similar?

**Comment by Floppy** [comment]
Yes, we should add this under privacy. For URLs, the data is not stored locally, just read off the web via the public URL. The URL is included in our list of recent validations. For file uploads, the file is stored temporarily in order to allow validation, but is then cleaned out after 24 hours. Validations from file uploads are not listed in the public validation list.

Is that what you need?

**Comment by bcouston** [comment]
Yes that's great, I'm currently waiting on final confirmation on whether this issue is aimed towards users or developers.

**Comment by Floppy** [comment]
The API documentation is aimed at developers, but make sure the general stuff like privacy etc is aimed at a general user, so is included in a privacy policy page.

**Comment by bcouston** [comment]
@Floppy Quick question, are validations from uploaded files stored but not displayed in public validations or not stored at all?

**Comment by bcouston** [comment]
Added issue #706 as part of this.

**Comment by bcouston** [comment]
Wrote a list of features that are/are not mentioned in About and constructed my plan for adding documentation: https://docs.google.com/a/theodi.org/document/d/1g6jHlkxpoSaLDf-FV1DwYLbekv1LbN0CG03oHsUQ0oQ/edit?usp=sharing

**Comment by dicknorris** [comment]
Book to DaPaaS > WP1 > Task 1.2
